rate: 50
controllers:
  - setpointTopic: "testSetpointTopic0"
    feedbackTopic: "testFeedbackTopic0"
    outputTopic: "testOutputTopic0"
    P: 9
    I: 1.25
    D: 0.03
    min: -100
    max: 100

# Extra controller entries go here